src/libs/GrowlNotification/GrowlNotificationContainer/GrowlNotificationContainerPropTypes.js

```diff
@@ -0,0 +1,15 @@
+import {Animated} from 'react-native';
+import PropTypes from 'prop-types';
+import {windowDimensionsPropTypes} from '../../../components/withWindowDimensions';
+
+const propTypes = {
+    // GrowlNotification content
+    children: PropTypes.node.isRequired,
+
+    // GrowlNotification Y postion, required to show or hide with fling animation
```

Comment:
Thanks, we are using block style `/** */` not `//` examples here 

https://github.com/Expensify/Expensify.cash/blob/main/STYLE.md#method-naming-and-code-documentation

Reply:
Okay, I changed the code

Reply:
My bad @marcaaron, I've got to remember this going forward!